conda_metadata_app/app_config.py

```diff
@@ -428,22 +427,5 @@ def export_json_schema() -> None:
         f.write("\n")
 
 
-def app_version() -> str:
-    try:
-        return check_output(
-            [
-                "git",
-                "--no-pager",
-                "log",
-                "-1",
-                "--pretty=format:`%h (%cd)`",
-                "--date=format:%Y-%m-%d %H:%M:%S",
-            ],
-            text=True,
-        ).strip()
-    except CalledProcessError:
-        return ""
```

Comment:
We have lost this branch now, so if there's an error fetching the version from git, it will reach the Streamlit app.

Reply:
Do we really need this? I think the only supported deployments are streamlit cloud or the Docker image anyway

Reply:
I don't know where people are deploying this or whether they have `git` available. The failsafe is required.

Reply:
Thanks!